op-node/rollup/derive/calldata_source.go

```diff
@@ -104,21 +120,28 @@ func (ds *DataSource) Next(ctx context.Context) (eth.Data, error) {
 // DataFromEVMTransactions filters all of the transactions and returns the calldata from transactions
 // that are sent to the batch inbox address from the batch sender address.
```

Comment:
Update the comment as now we will have multiple batchers.

Reply:
I think it may be best to keep track of the outdated comments somewhere first and then update them once, when we're happy with our design.